### PR TITLE
General overhaul of DK systems. Minor bugs left in OS and DM_ED50

### PR DESCRIPTION
**Pull request by busstoptaktik**
With much input from Karsten Engsager, we now have self tests and definitions for a large number of current and legacy Danish systems.

The work on this PR revealed a few remaining bugs in the current PROJ release candidate. It will first run smoothly once my patch for those has been applied over at PROJ.

### CODE REVIEW

File: resources/DK [hunk 1]
```diff
@@ -1,74 +1,718 @@
...
+#------------------------------------------------------------------------------------------------------
+#  These formal definitions are for illumination - not for computation.
+#  Valid only for observations already measured in the relevant (legacy)
+#  reference frame.
+#
+#  See below for the much more involved setups needed to go to and from
+#  the legacy frames.
+#------------------------------------------------------------------------------------------------------
+# UTM ED50
+<UTM32_ED50_FORMAL> proj=utm zone=32 ellps=intl units=m no_defs
```
Comment (kbevers) [inline resources/DK]: This really breaks the core idea of this init-file using ETRS89 as pivot datum. I do appreaciate the need to have a simple version of the legacy system. I'd suggest adding another init file called `ED50` and make that a shared init file for all legacy systems based on ED50 in the Nordic countries. Each country specific init file can then include a mapping to ED50. We can then do something like:

```
proj=pipeline +step +init=DK:ED50 +step +init=ED50:SB
```
Which will transform a ETRS89 geodetic coordinate to the Storebælt system (based on ED50).

Assuming that the existing polynomial mappings allow such a setup.

**Comment by kbevers** [comment]
Can this be merged? The windows is open for adding init-files to the proj-datumgrids package and I am keen to get this in before the next version is released.

**Comment by busstoptaktik** [comment]
Can it wait until Monday afternoon? I have added a number of improvements in my local branch, but not yet tested fully - and still missing metadata

**Comment by kbevers** [comment]
Yes.

**Pull request merged**